drivers/kscan/kscan_input.c

```diff
@@ -100,18 +100,15 @@ static const struct kscan_driver_api kscan_input_driver_api = {
 	.disable_callback = kscan_input_disable_callback,
 };
 
-#define KSCAN_INPUT_INIT(index) \
-	INPUT_CALLBACK_DEFINE(DEVICE_DT_GET(DT_INST_PARENT(index)),            \
-			      kscan_input_cb,                                  \
-			      (void *)DEVICE_DT_INST_GET(index));              \
-	static const struct kscan_input_config kscan_input_config_##index = {  \
-		.input_dev = DEVICE_DT_GET(DT_INST_PARENT(index)),             \
-	};                                                                     \
-	static struct kscan_input_data kscan_input_data_##index;               \
-	DEVICE_DT_INST_DEFINE(index, kscan_input_init, NULL,                   \
-			      &kscan_input_data_##index,                       \
-			      &kscan_input_config_##index,                     \
-			      POST_KERNEL, CONFIG_KSCAN_INIT_PRIORITY,         \
-			      &kscan_input_driver_api);
+#define KSCAN_INPUT_INIT(index)                                                                    \
+	INPUT_CALLBACK_DEFINE_NAMED(DEVICE_DT_GET(DT_INST_PARENT(index)), kscan_input_cb,          \
+				    (void *)DEVICE_DT_INST_GET(index), kscan_input_cb_##index);    \
+	static const struct kscan_input_config kscan_input_config_##index = {                      \
+		.input_dev = DEVICE_DT_GET(DT_INST_PARENT(index)),                                 \
+	};                                                                                         \
+	static struct kscan_input_data kscan_input_data_##index;                                   \
+	DEVICE_DT_INST_DEFINE(index, kscan_input_init, NULL, &kscan_input_data_##index,            \
+			      &kscan_input_config_##index, POST_KERNEL,                            \
+			      CONFIG_KSCAN_INIT_PRIORITY, &kscan_input_driver_api);
```

Comment:
Unfortunately clang-format destroyed the diff. My only change was
```diff
- INPUT_CALLBACK_DEFINE(DEVICE_DT_GET(DT_INST_PARENT(index)), kscan_input_cb, (void *)DEVICE_DT_INST_GET(index));
+ INPUT_CALLBACK_DEFINE_NAMED(DEVICE_DT_GET(DT_INST_PARENT(index)), kscan_input_cb, (void *)DEVICE_DT_INST_GET(index), kscan_input_cb_##index);
```

Reply:
yeah it makes a bit of a mess, actually would it bother you too much to change this and the previous patch to minimize the diff and not use clang-format? just intend them manually and keep them on 80 columns

not a clang-format fun myself